_posts/2017-11-27-overcoming-language-learning-frustrations-guest-post-by-lingua-pony.md

```diff
@@ -0,0 +1,69 @@
+perhaps, the hardest to deal with. It is, of course, very easy to say, ‘don’t be jealous of your fellow learners, but use their 
+example to motivate yourself. We all have different abilities and different pace, but we will all arrive at the end if we work hard’.
+
+Obvious things, but, alas, easier said than done. For someone lagging continuously behind the others these truisms may in fact be 
+of no help, even annoying. What will really work is to switch to individual lessons, assuming that they are affordable, but constant 
+attention on the part of the teacher, not shared with other students, can be too much for some. Whenever it’s possible to create 
+a mini-group, in which there would be two or three students carefully matched by their abilities, that would be a perfect solution.  
+
+Some learners get frustrated if the language they have chosen has turned out to be too difficult. The majority of such complaints 
+are about Mandarin Chinese, with Hungarian and Russian following close behind. What can I say to that? Russian is easy for me, because 
+it’s my native language, but Mandarin is indeed a challenge. When at times I feel like giving it up, I remind myself that a lot 
+of people have mastered it. It may have taken a lot of time, but it is doable. And that means I can do it too, so I just persevere.
+
+But in some cases it might be wise to try another language if the one you have chosen is a total no-go. There are thousands 
+of them around waiting for you to pay attention, and no one has learned them all so far, so, unless it is of the utmost importance 
+for you to master the language you are struggling with – e.g. for your professional needs – it’s okay to shift your attention to 
+something more agreeable.
+
+But there is yet another and especially disturbing reason of language learning frustrations – and that’s meanness of people around us. 
+Sometimes it’s a teacher freaking out and saying all sorts of nasty things to a student for no reason. Sounds fantastic? Well, it does 
+happen; it has happened. The only way to deal with such a teacher is to fire him or her at once, but still it can be too late because 
+harm has been done. Also, if you seek feedback at online forums, make sure you pick a friendly, supportive community, in which your 
+fellow members will be aiming at helping you, as opposed to building up their self-esteem by means of destroying yours. Evil things 
+do happen online, unfortunately, since being on the web gives many a sense of impunity; once get burned by such an experience, some of 
+us hurt for months, if not for years. 
+
+Should we meet a person whose self-confidence has been harmed by someone else’s nastiness, it’s our duty to offer moral support. 
+Language learning community should be a brotherhood – not a battlefield. Sometimes it’s enough just to tell an insecure person that 
```

Comment:
Let's not forget the sisters, too, who are too often forgotten! :-) 

(I actually think that female language learners might be the majority, but the guys tend to get more attention.)